### PR TITLE
Update to stack and client 8.2

### PR DESCRIPTION
**Pull request by jsanz**
Updates EMS Client and points to manifests for 8.2 version.

### CODE REVIEW

File: .ci/jobs/defaults.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@
           &lt;commitId&gt;, etc.)
     - string:
         name: root_branch
-        default: 'v8.0'
+        default: 'v8.2'
```
Comment (nickpeihl) [inline .ci/jobs/defaults.yml:22]: IIRC, this sets the default branch for https://kibana-ci.elastic.co/job/elastic+ems-landing-page+deploy/build?delay=0sec. But we can specify a different branch when running the deploy build. So until 8.2 is released, we would change the root_branch to 8.1 when running the deploy build. That's fine with me.

**Review by nickpeihl — APPROVED** [review]
lgtm! code review and tested in firefox

File: .ci/jobs/defaults.yml (hunk 1 above)
Comment (jsanz) [reply .ci/jobs/defaults.yml:22]: sure, I'll be sure to run the deploy pointing to 8.1 for now. Thanks for the pointer!

**Comment by jsanz** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------:|
|✅|v8.2|[](https://github.com/elastic/ems-landing-page/pull/421)|

**Pull request merged**